packages/uows-client-generator/src/index/index.ts

```diff
@@ -40,10 +40,12 @@ const generateCode = (obj: any, wsdl: string, filePath: string): void => {
     `
     import * as soap from 'soap';
     import { logger } from '@user-office-software/duo-logger';
+    import { createHash } from 'crypto';
 
     export default class UOWSSoapClient {
       private wsdlUrl: string;
       private client!: soap.Client;
+      private activeUowsRequests: Record<string, Promise<any>> = {};
```

Comment:
Don't know if it'd be significant, but I feel like map entries might be more... suited(?) than object properties.

Reply:
I actually didn't know that JavaScript has Maps now. I was stuck in the old days when object properties were the only option for this sort of thing